packages/esm-patient-registration-app/translations/en.json

```
@@ -15,7 +15,7 @@
   "configure": "Configure",
   "configureIdentifiers": "Configure identifiers",
   "contactSection": "Contact Details",
-  "createNew": "Create New",
+  "createNewPatient": "Create New Patient",
```

```suggestion
  "createNewPatient": "Create new patient",
```

Prefer sentence-case. See https://zpl.io/VDNg9BO and https://carbondesignsystem.com/guidelines/content/writing-style#capitalization.